src/BloomExe/Program.cs

```diff
@@ -860,5 +859,30 @@ private static void CheckLinuxFileAssociations()
 			Debug.Print("Executing update-desktop-database");
 			proc.Start();
 		}
+
+		/// <summary>
+		/// Getting the count of running Bloom instances takes extra steps on Linux.
+		/// </summary>
+		/// <returns>The number of running Bloom instances</returns>
+		public static int GetRunningBloomProcessCount()
+		{
+			var i = Process.GetProcesses().Count(p => p.ProcessName.ToLower().Contains("bloom"));
```

Comment:
It would be clearer to use `n` here instead of `i` since it's not an index. Or make it more descriptive with something like `bloomProcessCount`.